### ISSUE TITLE
Release Twisted Web2 0.2

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1810]
|[<img alt="dreid's avatar" src="https://avatars.githubusercontent.com/u/48695?s=50" width="50" height="50">](https://github.com/dreid)| @dreid reported|
|-|-|
|Trac ID|trac#1810|
|Type|enhancement|
|Created|2006-06-09 23:13:05Z|

Release rar!

<details><summary>Searchable metadata</summary>

```
trac-id__1810 1810
type__enhancement enhancement
reporter__dreid dreid
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web2 web2
keywords__ 
time__1149894785000000 1149894785000000
changetime__1150004443000000 1150004443000000
version__None None
owner__ 
cc__edsuom
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="dreid's avatar" src="https://avatars.githubusercontent.com/u/48695?s=50" width="50" height="50">](https://github.com/dreid)|@dreid set owner to PenguinOfDoom|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="dreid's avatar" src="https://avatars.githubusercontent.com/u/48695?s=50" width="50" height="50">](https://github.com/dreid)|@dreid commented|
|-|-|

[branches/releases/twisted-web2-0.2.x-1810-2](branches/releases/twisted-web2-0.2.x-1810-2)

**Comment by twisted-trac** [comment]
|<img alt="PenguinOfDoom's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|PenguinOfDoom set owner to @dreid|
|-|-|

Looks fine.

**Comment by twisted-trac** [comment]
|[<img alt="dreid's avatar" src="https://avatars.githubusercontent.com/u/48695?s=50" width="50" height="50">](https://github.com/dreid)|@dreid set status to `closed`|
|-|-|

(In [17179]) Merge releases/twisted-web2-0.2.x-1810-2.

Author: dreid
Reviewer: PenguinOfDoom
Fixes [#1810](https://github.com/twisted/twisted/issues/1810)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
